index.js

```diff
@@ -1,9 +1,19 @@
-//main file for our application
-import controller from "./src/controllers/sampleController.js";
+import assignAgenttoTicketPart2 from './assignAgenttoTicketPart2.js';
+
+const sleep = ms => new Promise(r => setTimeout(r, ms));
+
+const app2 = async () => {
+
+
+    let agentLastName = '';
+
+    while (1 == 1) {
+
+        agentLastName = await assignAgenttoTicketPart2(agentLastName);
+        console.log(agentLastName);
+        await sleep(1000);
+    }
```

Comment:
while should work, but I think we were discussing to use some 'nicer' solution like scheduler (e.g. cron) - it can stay as it is though